strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/layout/AbstractLayoutProvider.java

```diff
@@ -465,8 +463,7 @@ protected void doDeletePath(RepositoryPath repositoryPath,
     {
         Files.delete(repositoryPath);
 
-        Repository repository = repositoryPath.getFileSystem()
-                                              .getRepository();
```

Comment:
If it's a long list of these, sure, this should be chunked and aligned by the first method call and then the chained ones should all be on new lines. But... when it's a handful of short-named methods that fit on the same line, it looks odd to take 4-5 lines for something like:

    foo.bar()
       .do()
       .it();

instead of just:

    foo.bar().do().it();

If you guys feel strongly about it, then sure, we can adopt this. But I just don't want to see a lot of unnecessary code reformatting in pulls outside the blocks which are being worked on.